Makefile

```diff
@@ -44,6 +44,8 @@ servewallet-regtest:
 	go install ./cmd/servewallet/... && servewallet -regtest
 servewallet-multisig:
 	go install ./cmd/servewallet/... && servewallet -multisig
+servewallet-prod:
```

Comment:
maybe `-prodservers` to indicate it's only about this?

Reply:
Yup.